.github/workflows/pr-main_l1.yaml

```diff
@@ -82,11 +82,13 @@ jobs:
         working-directory: tooling
 
   test:
-    # "Test" is a required check, don't change the name
-    name: Test
+    name: Test - ${{ matrix.runner }}
     needs: detect-changes
     if: ${{ needs.detect-changes.outputs.run_tests == 'true' }}
-    runs-on: ubuntu-latest
+    runs-on: ${{ matrix.runner }}
+    strategy:
+      matrix:
+        runner: [ubuntu-24.04, ubuntu-24.04-arm, macos-15]
```

Comment:
We are using Ubuntu 22.04 in release workflows to be compatible with Debian 12's glibc. Should we use the same here?